### PR TITLE
The dev version works fine at this point

### PR DESCRIPTION
**Pull request by ReturnOfTheLast**

### CODE REVIEW

**Comment by ReturnOfTheLast** [comment]
Please review

**Review by ReturnOfTheLast — COMMENTED** [review]
I wrote must of it so I hope it is fine

File: src/LoRa/LoRaTDMAMacFrame.msg [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+cplusplus {{
+using namespace inet;
+}}
+
+namespace flora_tdma;
+
+class LoRaTDMAMacFrame extends inet::FieldsChunk {
+    inet::MacAddress transmitterAddress;
+    // inet::MacAddress receiverAddress;
```
Comment (Mikkel-Coder) [inline src/LoRa/LoRaTDMAMacFrame.msg:29]: We should remove this if we got time

File: src/NetworkServer/NetworkServerApp.h
Comment (Mikkel-Coder) [inline src/NetworkServer/NetworkServerApp.h:1]: I think we should remove this and the other modules, such as the "internet ppp delay thing" as we do not use them

**Review by Mikkel-Coder — APPROVED** [review]
I think we will remove the unused code in the future. LGTM!

**Comment by Mikkel-Coder** [comment]
Please approve in the future @MzKay-chan

**Pull request merged**